backoff/job.go

```diff
@@ -0,0 +1,39 @@
+package backoff
+
+import (
+	"github.com/cenk/backoff"
+	"time"
+)
+
+var (
+	_ backoff.BackOff = (*JobBackOff)(nil)
+)
+
+const (
+	defaultMinJobInterval = 30 * time.Second
+)
+
+// JobBackOff is an exponential backoff implementation for long running jobs.
+// In long running jobs, an operation() that fails after a long Duration should not increments the backoff period.
+// If operation() takes more than MinJobInterval, Reset() is called in NextBackOff().
+type JobBackOff struct {
```

Comment:
(instead of `backoff` currently — which is why you need to do an alias as of this PR)